### DISCUSSION TITLE
Cant indent or outdent in visual editor or markdown

### DISCUSSION THREAD

**Discussion by arlorn** [requarks/wiki#6039] [Help / Questions]
I can't get an indent to work in the visual editor. Hitting Tab jumps to the table in the document or leaves the editor and interacts with the browser. "- " before text just creates a bulleted list. 
I'm trying to add structure to text without using 10 or 20 spaces before the line of text. It seems like the function exists in the documentation for [ckeditor](https://ckeditor.com/docs/ckeditor5/latest/features/indent.html), I'm clueless on how to get it to work in wiki.js. and any help is appreciated. 
I'm doing a linux hosted instance of 2.5.289 and I have limited linux knowledge (took me FOREVER to get this instance up and running due to being a windows guy).

**Comment by kiliw** [comment]
Same here. I tried adding it to the editor as in the documentation, but then everything is broken. Unfortunately, as stated here https://github.com/requarks/wiki/discussions/5485 there seems no easy way to add this.
